### PR TITLE
fix swagger for how to register a new user

### PR DESCRIPTION
**Pull request by GeraldLoeffler**
To register a new user POST to `/users` not to `/user`.

### CODE REVIEW

File: api/swagger.json [hunk 1]
```diff
@@ -104,7 +104,9 @@
             }
           }
         }
-      },
+      }
+    }
```
Comment (thebearingedge) [inline api/swagger.json:108]: There appears to be a comma missing here.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by GeraldLoeffler — COMMENTED** [review]
missing comma added

**Comment by GeraldLoeffler** [comment]
thank you @thebearingedge , and sorry for the hickup: should be good now

**Comment by geromegrignon** [comment]
Thanks for your contribution.
However it's a breaking change for all existing implementations.

To address such issue, your goal is to work on a V2 of the current specs.
Based of these new specs, we'll open the submission of a new generation of implementations.
The current version of the spec will remain unchanged.

We'll track this issue with #626.

**Pull request closed without merging**